examples/sample_identity_map_client.py

```diff
@@ -0,0 +1,38 @@
+import sys
+
+from uid2_client import IdentityMapClient, IdentityMapInput
+
+
+# this sample client takes email addresses or phone numbers as input and generates an IdentityMapResponse object
+# which contains raw uid or the reason why it is unmapped
+
+def _usage():
+    print('Usage: python3 sample_sharing_client.py <base_url> <auth_key> <secret_key> <email_list>', file=sys.stderr)
```

Comment:
`email_list` raises some questions (comma separated, quotes?)
Maybe we can give an example usage as well here.

Reply:
that is a good point, space is good

Reply:
I updated it to `Usage: python3 sample_sharing_client.py <base_url> <auth_key> <secret_key> <email_1> <email_2> ... <email_n>`